Core/Utilities/TimeLeft/test/Test_TimeLeft.py

```diff
@@ -53,13 +93,27 @@ def test_getScaledCPU( self ):
       self.tl.runCommand = rcMock
 
       batchSystemName = '%sTimeLeft' % batch
-      batchPlugin = __import__( 'DIRAC.Core.Utilities.TimeLeft.%s' %
+      batchPlugin = __import__( 'DIRAC.Core.Utilities.TimeLeft.%s' % #pylint: disable=unused-variable
                                 batchSystemName, globals(), locals(), [batchSystemName] )
       batchStr = 'batchPlugin.%s()' % ( batchSystemName )
       tl.batchPlugin = eval( batchStr )
       res = tl.getScaledCPU()
       self.assertEqual( res, 300.0 )
 
+      for batch, retValue in [( 'MJF', MJF_ReturnValue )]:
+        self.tl = importlib.import_module( "DIRAC.Core.Utilities.TimeLeft.TimeLeft" )
+        rcMock = MagicMock()
```

Comment:
@fstagni for just a question of preference, it does not really matter, but @andresailer has a point I did not think of. If you really need this PR in today, we can probably merge it as is, but please add a comment explaining how dangerous this is, and consider a different way of doing in a soon to come PR ?